.lightningignore

```diff
@@ -0,0 +1 @@
+.git
```

Comment:
Could this be removed  ?

Reply:
when I run the acceptance tests locally the entire lightning repository is packaged and uploaded to Lightning AI - maybe this is due to how I run things.
I'll remove it.